### PR TITLE
Fixing #112: Simulate: rename port option to simulatePort

### PR DESCRIPTION
**Pull request by dlebu**
#112

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -199,7 +199,7 @@
                         "request": "launch",
                         "platform": "android",
                         "target": "chrome",
-                        "port": 8000,
+                        "simulatePort": 8000,
```
Comment (MSLaguana) [inline package.json]: Should we also keep a definition of `port` as well, to show that both can apply?

Comment (dlebu) [inline package.json]: We could, for consistency with the other tasks.

**Comment by MSLaguana** [comment]
Looks good to me otherwise.

File: package.json (hunk 1 above)
Comment (dlebu) [inline package.json]: We ended up not adding it for the browser scenarios, for consistency with ionic serve scenario, which does not have it defined.

**Comment by dlebu** [comment]
Thanks for the feedback, merging to master!

**Pull request merged**